core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -292,33 +292,33 @@
    */
   public static final String HOTSPOT_OPN_FRACTION_DEFAULT = "0.8";
 	
-	IntegerGenerator keysequence;
+	LongGenerator keysequence;
 
 	DiscreteGenerator operationchooser;
 
-	IntegerGenerator keychooser;
+	LongGenerator keychooser;
 
 	Generator fieldchooser;
 
 	AcknowledgedCounterGenerator transactioninsertkeysequence;
 	
-	IntegerGenerator scanlength;
+	LongGenerator scanlength;
 	
 	boolean orderedinserts;
 
-	int recordcount;
+	long recordcount;
 
-    private Measurements _measurements = Measurements.getMeasurements();
+  private Measurements _measurements = Measurements.getMeasurements();
 	
-	protected static IntegerGenerator getFieldLengthGenerator(Properties p) throws WorkloadException{
-		IntegerGenerator fieldlengthgenerator;
+	protected static LongGenerator getFieldLengthGenerator(Properties p) throws WorkloadException{
```

Comment:
We're already using this patch to go from tables with MAX_LONG instead of MAX_INT. Let's save moving from individual fields being up to MAX_INT to MAX_LONG in a different patch.